apps/frontend/core-dapp/src/features/trade/open-trade/OpenTradeSummary.tsx

```diff
@@ -31,7 +32,8 @@ const OpenTradeSummary: React.FC = () => {
   const renderValueUI = (): React.ReactNode => {
     if (loading) return <Skeleton height="22px" width="64px" />
     const unitPrice = selectedMarket.getUnitPriceString(tradingValuation)
-    if (!withinBounds) return <RedText>Unprofitable</RedText>
+    if (withinBounds === false || ethWstEthPriceImpactTooHigh)
+      return <RedText>Unprofitable Trade</RedText>
```

Comment:
Pretty sure it should only say Unprofitable in the summary

```suggestion
    if (withinBounds === false || ethWstEthPriceImpactTooHigh)
      return <RedText>Unprofitable</RedText>
```